.mergify.yml

```diff
@@ -2,7 +2,8 @@ pull_request_rules:
   - name: automatic merge
     conditions:
       - base=master
-      - label=automerge
+      - label!=manual merge
```

Comment:
does github actually support whitespace in labels?

Reply:
of course. and unicode!